package-lock.json

```
@@ -6722,13 +6722,30 @@
 			}
 		},
 		"@wordpress/dependency-extraction-webpack-plugin": {
-			"version": "2.9.0",
-			"resolved": "https://registry.npmjs.org/@wordpress/dependency-extraction-webpack-plugin/-/dependency-extraction-webpack-plugin-2.9.0.tgz",
-			"integrity": "sha512-Eo8ByPd3iZ6az4UmdLD2xYLp1/7os/H80l28Y5OlS4DozkD3vcWCBReynWoBax74u3oJ9wWN5b/8oSxGwIKXYQ==",
+			"version": "https://gitpkg.now.sh/WordPress/gutenberg/packages/dependency-extraction-webpack-plugin?trunk",
```

💅 
Since the `trunk` of gutenberg is continuously updating, checksum failure occurs when running `npm install`, it'd be easier to test PR with the specified commit hash.

```suggestion
			"version": "https://gitpkg.now.sh/WordPress/gutenberg/packages/dependency-extraction-webpack-plugin?6d87c733c77c33f821fde94e5a8203212aec422b",
```